central/policy/service/validator.go

```diff
@@ -104,17 +105,24 @@ func (s *policyValidator) internalValidate(policy *storage.Policy, additionalVal
 	return errorList.ToError()
 }
 
-func (s *policyValidator) validateName(policy *storage.Policy) error {
-	return nameValidator.Validate(policy.GetName())
-}
-
 func (s *policyValidator) validateVersion(policy *storage.Policy) error {
-	if !policyversion.IsBooleanPolicy(policy) {
-		return errors.New("policy not converted to boolean policy")
+	ver, err := policyversion.FromString(policy.GetPolicyVersion())
+	if err != nil {
+		return errors.New("policy has invalid version")
+	}
+
+	// As of 70.0 we only support the latest version (1.1). This may in the future be expanded to support more, but for
+	// now it's enough to just check it matches current version
+	if !policyversion.IsCurrentVersion(ver) {
```

Comment:
Can we change this to `IsSupportedVersion()`? The supported versions need not be current version.